values/kubeclarity/kubeclarity.gotmpl

```diff
@@ -0,0 +1,14 @@
+{{- $v := .Values }}
+{{- $k := $v.apps.kubeclarity }}
+
+global:
+  databasePassword: {{ $k | get "databasePassword" }}
+
+kubeclarity-postgresql:
+  fullnameOverride: kubeclarity-postgresql
+
+kubeclarity:
+  logLevel: {{ $k | get "logLevel" "warning" }}  
```

Comment:
since the  default.yaml file already specifies the value then simply refer as `$k.logLevel`